docs/docs/conferences.md

```diff
@@ -0,0 +1,19 @@
+﻿---
```

Comment:
There's actually a zerowidth space at the beginning of this line which is tripping up jekyll. Can you re-save this file and make sure it goes away?

You can actually see this on GitHub too - the front matter here should get parsed into a table. Compare https://github.com/rickbeerendonk/react/blob/docs-conferences/docs/docs/videos.md and https://github.com/rickbeerendonk/react/blob/docs-conferences/docs/docs/conferences.md